src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const basicCost = 40;
+  const basicDiscount = 20;
+  const biggerDiscount = 50;
+
+  if (days < 3) {
+    return (days * basicCost);
+  }
+
+  if (days >= 3 && days < 7) {
+    return (days * basicCost - basicDiscount);
```

Comment:
You don't need brackets in this place
```suggestion
    return days * basicCost - basicDiscount;
```